src/resources/js/tribe-events-bar.js

```diff
@@ -393,3 +333,426 @@ var tribe_events_bar_action;
 
 })( window, document, jQuery, tribe_ev.data, tribe_ev.events, tribe_ev.fn, tribe_ev.state, tribe_ev.tests, tribe_debug );
 
+(function( factory ) {
```

Comment:
It is a bummer that Select2 is not yet at the level of accessibility support that this plugin is. Adding another select box implementation into the mix is frustrating.